### PR TITLE
Add CODEOWNERS

### PR DESCRIPTION
**Pull request by ryjones**
Signed-off-by: Ry Jones <ry@linux.com>

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Indy CI
+*	@evernym-ci @nage @sovbot
+
+# Indy Common
+*	@anikitinDSR @ashcherbakov @brentzundel @devin-fisher @dhh1128 @glowkey @keenondrums @lovesh @michaeldboyd @mikelodder7 @nage @rajeshkalaria80 @Toktar @vimmerru
```
Comment (ashcherbakov) [inline CODEOWNERS]: I believe  @skhoroshavin is missing here.

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Indy CI
+*	@evernym-ci @nage @sovbot
```
Comment (ashcherbakov) [inline CODEOWNERS]: We may add @skhoroshavin  and @anikitinDSR to Indy CI list.

**Comment by ryjones** [comment]
I removed the INDY-CI section for now.

**Comment by dhh1128** [comment]
(ci) test this please

**Comment by ashcherbakov** [comment]
(ci) test this please

**Pull request merged**